### PR TITLE
Cures zombies

### LINKED ISSUE #12690: Latejoin tators can get an objective to assassinate simple mobs (maybe zombies in specific only)
Marflow was Olorin McKellen the Virologist and had their body destroyed
Objective #1: Assassinate zombie, the Animal. Success!
Objective #2: Escape on the shuttle or an escape pod alive and without being in custody. Fail.
The traitor has failed!

### LINKED ISSUE #15194: Zombie species and simple mob issues.
So right now if you create a simple mob zombie, upon killing their target they will also become a zombie (nice!) also if they die, they will be resurrected in a few minutes. 
BUT if you turn a player into a zombie (via Set Species in VV) they don't have the ability to transmit the "disease" nor will they resurrect, they simply LOOK like a zombie.

Is there anyway to change this?

Also it be nice if both zombie types couldn't speak plain human, is it possible to either create a new language that allows them to talk among themselves and to only come out as  Mumbles or "BRAIIINNNS" or something similar. Right now its pretty immersion breaking when a zombie can scream "Get back here you fgt" etc etc... (yes Ive seen it happen)

### LINKED ISSUE #17932: Butchering zombies produces a "the infection holder bursts out of X" message
The INVISIBLE_ABSTRACT passive simple_animal zombie is there! I CAN SMELL IT.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Halloween is over I'm sorry for inflicting these on the codebase please forgive me.

If this upsets too many people we can instead just remove them from xenobio and the lavaland gym and admins can drag them out for """""events""""" (preferably in october)

Cheriedit: Closes #17929
Effectively Fixes #17932
Effectively Fixes #12690
Effectively Fixes #15194

### CODE REVIEW

**Comment by coiax** [comment]
Nice meme. Hey remember when you wanted to remove lavaland

**Comment by Ultimate-Chimera** [comment]
:+1:

**Comment by Cheridan** [comment]
@coiax lavaland isn't cancer zombies are

File: _maps/RandomRuins/LavaRuins/lavaland_gym.dmm [hunk 1]
```diff
@@ -1,53 +0,0 @@
-"a" = (/turf/closed/mineral/volcanic/lava_land_surface,/area/lavaland/surface/outdoors)
```
Comment (Iamgoofball) [inline _maps/RandomRuins/LavaRuins/lavaland_gym.dmm]: NO

**Comment by MMMiracles** [comment]
I mean I could see removing the ruin but zombies entirely?

wheres that copypasta with the officer nofun

**Comment by Iamgoofball** [comment]
# DO NOT REMOVE THE GYM PLS

## if you dont want zombies in it I can swap it for people

**Comment by KorPhaeron** [comment]
Comparing lavaland to zombies is very depressing.

**Comment by Cheridan** [comment]
beach already has gym eqquipment and it's way better mapped so succ it goofball

**Comment by MMMiracles** [comment]
i mean yeah tbh the gym is pretty ugly looking but zombies are cool just leave them for admin memery

**Comment by Iamgoofball** [comment]
@Cheridan WHAT WHY

THE EQUIPMENT WAS SUPPOSED TO BE ONLY AVAILABLE IN THE GYM, WTF ARE YOU FAGGOTS DOING

**Comment by Zilenan91** [comment]
Just take the zombies out of everything but xenobio. Xenobio is in a fantastic spot right now as far as fun and balance goes.

**Comment by PKPenguin321** [comment]
this is kinda like removing xenos or spiders

:-1:

**Comment by KingDairy** [comment]
fuck♂you

**Comment by Cheridan** [comment]
@PKPenguin321 exactly both those things are fucking awful so removing them is great

**Comment by Steelpoint** [comment]
I like zombies, they just need some more love.

**Comment by Ultimate-Chimera** [comment]
Zombies ruin rounds, I had a changeling round ruined because people abuse them. :+1: remove.

**Comment by Niknakflak** [comment]
Blame robustin for this, dude loves his zombies

**Comment by Steelpoint** [comment]
No I think removing Zombies outright is bad, they can be enjoyable and fun but in moderation.

I think the core issue is the presence of Zombies on Lavaland. I think the better outcome is removing zombies from lavaland and making Zombies a admin event only for now.

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 2]
```diff
@@ -1,202 +0,0 @@
...
-	stat_attack = 2
-	gold_core_spawnable = 1
-	faction = list("zombie")
-	var/mob/living/carbon/human/stored_corpse = null
-	butcher_results = list(/obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie = 3)
-	see_invisible = SEE_INVISIBLE_MINIMUM
-	see_in_dark = 8
-	layer = MOB_LAYER - 0.1
-	var/removingairlock = 0
-
```
Comment (pudl) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: just comment it out

**Comment by LatD** [comment]
Zombies are fun and they've received balances to not be too strong and crew has (somewhat) learned how to fight against them. They're usually round enders like xenos, spiders and swarmers. Just make them more rare and remove them from ruins (if that's where they're coming from lately).

File: code/modules/mob/living/simple_animal/hostile/zombie.dm (hunk 2 above)
Comment (Steelpoint) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: Don't even need to comment it out, just remove the zombie lavaland ruin.

**Comment by Robustin** [comment]
Zombies are great, might need 20 more HP off or -5 damage but they're a lot better than they used to be and they only started to become an "issue" when lavaland gyms were showing up in 30% of rounds, throw in all the shit on lavaland that can get zombified and zombie hordes showing (and me getting blamed every time, FFS I only spawn them on extended) up is probably a little too common as a result. 

This PR is classic Kor. There can't be tweaks or slow, disciplined progress on balance it's always "Do nothing" or "Burn it to the fucking ground, we're gonna go fucking scorched earth on this shit".

**Comment by KorPhaeron** [comment]
https://www.tgstation13.org/phpBB/viewtopic.php?f=9&t=6751

>why wont anyone put their foot down on stupid shit and give us some direction? Typical coders

https://www.tgstation13.org/phpBB/viewtopic.php?f=10&t=6466&start=50#p172098

>why are you making shitty compromises? Nobody is happy with that! Typical coders

And now we have completed the trifecta with "why wont you compromise and keep a lazy redundant conversion antag in the game"

I'm sure you are taking this personally because you worked on them, but I still consider zombies my own embarassment and my fault.

**Comment by PKPenguin321** [comment]
I think they can be fun though. Don't remove zombies from the game, leave them as an admin only thing at least.

**Comment by KorPhaeron** [comment]
I was about to offer that as an option when I remembered I already did that in the op about 15 hours before the post about how I never compromise got made.

**Comment by lzimann** [comment]
zombies are boring as fuck

**Comment by PKPenguin321** [comment]
I disagree

**Comment by Robustin** [comment]
One of the best rounds I had in the past month was a rev round where zombies came from mining. Security had to stick together, clear the halls, avoid rev ambushes, hold off the undead, etc.

They definitely have potential but I understand and appreciate that when I was mining like 50% of rounds either had a zombie gym or that cursed shovel ruin, and maybe 25% of those rounds would result in someone bringing zombies/shovel back onto station and starting an outbreak. 10-15% of rounds being seriously impacted by zombies is a bad thing. Xenobio zombies are rarely an issue since:

1) the scientist is responsible for spawning/managing them safely, antag zombie outbreaks are ~emergent~ gameplay and probably more entertaining than someone just spamming gold cores/hostile slimes/spiders/etc. instead. 

2) 1-2 zombies are typically very easy to manage unless the halls are lined with corpses

This is a Lavaland issue just like Ashwalkers were. We didn't remove Ashwalkers and now they're generally something worth keeping around. Same deal applies to zombies. If you just want to gut a conversion-based "sideantag" then spiders and jungle fever should be first in line, not zombies.

**Comment by optimumtact** [comment]
They hit very hard and yet are surprisingly robust.

Either give them less health since they can respawn or make them slower.

**Comment by PKPenguin321** [comment]
Literally throw floor tiles at them you baddies

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 3]
```diff
@@ -33,7 +33,7 @@
 	layer = MOB_LAYER - 0.1
 	var/removingairlock = 0
 
-/mob/living/simple_animal/hostile/zombie/darkholme //BOY♂NEXT♂RUIN
+/mob/living/simple_animal/hostile/zombie/darkholme //BOY?NEXT?RUIN
```
Comment (KazeEspada) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: someones editor doesnt support unicode.

**Comment by GunHog** [comment]
I agree with Robstin here. Zombies are quite interesting and fun in my opinion. Even if I die to one, I will have the chance to play as well. I enjoy things like this which does not entirely boot you from the round. I am certain they can be adjusted if they are too powerful.

**Comment by Steelpoint** [comment]
All good, since this just removes the zombies from lavaland as such.

They just need time to be adjusted, and admins can still use them.

**Comment by GunHog** [comment]
Being admin only is asymptotically removal, given the rarity.

**Comment by Steelpoint** [comment]
Nevertheless I think after the feature freeze I would wager money on seeing PRs going up to work on the zombie concept.

**Pull request merged**